go.mod

```diff
@@ -72,3 +72,8 @@ require (
 	gopkg.in/yaml.v2 v2.3.0 // indirect
 	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
 )
+
+replace (
```

Comment:
This should be changed as soon as https://github.com/liftbridge-io/liftbridge-api/pull/57 and https://github.com/liftbridge-io/go-liftbridge/pull/122 are merged